test_scripts/API/PerformAudioPassThru/027_ATF_P_PerformAudioPassThru_UI_timeout_TTS_success.lua

```diff
@@ -0,0 +1,151 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [GeneralResultCodes] GENERIC_ERROR watchdog timeout from HMI
+-- SDL must return GENERIC_ERROR result to mobile app in case one of HMI components does not respond being supported and active
```

Comment:
@maria-mih, please check is this requirement, because I manage to find only CRQ with such name. In this CRS the previous requirement is linked.